addons/mail/static/src/messaging/entity/chatter/chatter.js

```diff
@@ -277,6 +277,9 @@ function ChatterFactory({ Entity }) {
         isActivityBoxVisible: attr({
             default: true,
         }),
+        isAside: attr({
```

Comment:
Maybe we can change later but in my way of thinking, it can also be used to change other things like chatter size for example. For now this is done by a class toggled (eww) in form renderer as part of compatibility layer but maybe it can be done later with `isAside` props. But maybe I'm overthinking this

Reply:
The prop name needs to be descriptive of what it will do for the component itself, and not descriptive of the context on which the parent will call it.

If more than border is done, maybe it should be "hasAlternativeStyle" or "isBoxed", or ...?

Reply:
If one name for all is too hard, maybe it can also be multiple props. Eg. if we have one prop for border config and one more prop for fixed width config, it's good too, even if both are always toggled at the same time in practice.